timesketch/lib/sigma_util.py

```diff
@@ -245,3 +258,61 @@ def get_sigma_rule(filepath, sigma_config=None):
             {'file_relpath':file_relpath})
 
         return rule_return
+
+def get_sigma_rule_by_text(rule_text, sigma_config=None):
```

Comment:
input validation needed, because user is uploading text that could contain un-expected content.

Reply:
there is already ```yaml.safe_load_all```used, so not sure how much more can be done / needed to be done

Reply:
@kiddinn  WDYT?

Reply:
well, since you use safe load, then you are only parsing standard YAML tags and no arbitrary python objects, so that should suffice in this case.